### PR TITLE
Python action

### PR DESCRIPTION
**Pull request by ioana-blue**
Add native support for python actions.

### CODE REVIEW

File: catalog/samples/hello.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import sys
+def main(dict):
+    if 'user' in dict:
```
Comment (rabbah) [inline catalog/samples/hello.py]: the swift, and java examples use 'name' as the property rather than 'user'.

File: catalog/samples/hello.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        print("hello, " + dict['user'] + "!")
+        return {"user":dict['user']}
```
Comment (rabbah) [inline catalog/samples/hello.py]: the swift and java examples return { greeting: "Hello ..." }

File: catalog/samples/hello.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    else:
+        print("hello world!")
+        return {"user":"world"}
```
Comment (rabbah) [inline catalog/samples/hello.py]: the swift and java examples return { greeting: "Hello stranger!" } in the else branch.

**Comment by ioana-blue** [comment]
fixed.

**Comment by rabbah** [comment]
Rebased and merges as [0dd5579](https://github.com/openwhisk/openwhisk/commit/0dd5579b2b4417c276410ccf46396ef61a16c486).

**Pull request closed without merging**